### ISSUE TITLE
BUG: Current lyrics get cropped

### ISSUE THREAD

**Issue by danielepintore** [BLeeEZ/amperfy#295]
This bug is present in the last commit of the main branch 55bd8c7.
The issue is that the current lyrics line looses some words because it get cropped when selected instead of having 2 lines.
[Here](https://github.com/user-attachments/assets/8f248e2e-d1b1-4787-aa0b-60eee84e83d3)  is a video of the issue.
The response that i get from the server to get the lyrics is this:
```
{
  "subsonic-response": {
    "status": "ok",
    "version": "1.15.0",
    "type": "gonic",
    "serverVersion": "0.16.4",
    "openSubsonic": true,
    "lyricsList": {
      "structuredLyrics": [
        {
          "lang": "xxx",
          "synced": true,
          "line": [
            {
              "start": 1850,
              "value": " Ah, yes"
            },
            {
              "start": 4190,
              "value": " Non li fermi, non li fermi, non li fermano più"
            },
            {
              "start": 6780,
              "value": " Meddaman e Goediman"
            },
            {
              "start": 8390,
              "value": " Non li fermi, non li fermi, non li fermano più"
            },
            {
              "start": 11080,
              "value": " Microspasmi Crew"
            },
            {
              "start": 12810,
              "value": " Non li fermi, non li fermi, non li fermano più, e yo"
            },
            {
              "start": 17230,
              "value": " Non ci trovi da tutte le parti, ma dalla stessa"
            },
            {
              "start": 19480,
              "value": " Di quello che fai tu con la tua crew non mi interessa"
            },
            {
              "start": 22060,
              "value": " Se tu hai la merda che pressa bella"
            },
            {
              "start": 23860,
              "value": " Ma non ti aspettare che io ti presenti a mia sorella"
            },
            {
              "start": 26110,
              "value": " Troppi scoppi e scarabocchi, qui ti rigano il palato e gli occhi"
            },
            {
              "start": 28740,
              "value": " Dicono di fare fiocchi 'sti farlocchi"
            },
            {
              "start": 30130,
              "value": " Ma si fanno come ciocchi con due ciocchi"
            },
            {
              "start": 32070,
              "value": " Tristi come un bimbo coi balocchi tutti rotti"
            },
            {
              "start": 34210,
              "value": " Dieci notti, son finiti, dove siete spariti?"
            },
            {
              "start": 36860,
              "value": " Il terremoto vi ha piazzato giusto tra i detriti"
            },
            {
              "start": 39290,
              "value": " Tu che cosa dici, dimmi tu che cosa pensi"
            },
            {
              "start": 41340,
              "value": " Da quale parte stai, in mezzo rischi investimenti"
            },
            {
              "start": 43590,
              "value": " Ti servono le lenti per cercare i basamenti"
            },
            {
              "start": 45650,
              "value": " O te l'inventi e quando sbagli poi ti penti"
            },
            {
              "start": 48160,
              "value": " Qui rischi contatti violenti"
            },
            {
              "start": 49920,
              "value": " Ma se non ti spaventi poi capisci i movimenti"
            },
            {
              "start": 52380,
              "value": " Scusi, signorina, lei lo sa come funziona?"
            },
            {
              "start": 54580,
              "value": " Certo che lo so, c'è il dj che li suona"
            },
            {
              "start": 56100,
              "value": " Medda e Geodi Microspasmi in zona"
            },
            {
              "start": 58910,
              "value": " Portano il viaggio nel villaggio e l'aria buona"
            },
            {
              "start": 60700,
              "value": " E non li fermi, non li fermi, non li fermano più"
            },
            {
              "start": 62930,
              "value": " Non li fermi anche se chiami mille macchine blu"
            },
            {
              "start": 65290,
              "value": " Porta il vero Microspasmi questa resta la crew"
            },
            {
              "start": 67370,
              "value": " Con la forma più la firma quando arrivano giù"
            },
            {
              "start": 69430,
              "value": " E non si pensi che si è spenti"
            },
            {
              "start": 72040,
              "value": " Abbiamo la materia ed occhi grossi densi"
            },
            {
              "start": 73950,
              "value": " E tu, tu che cosa pensi?"
            },
            {
              "start": 76110,
              "value": " Che alla fine resteranno solamente movimenti"
            },
            {
              "start": 78520,
              "value": " Di cosa ti lamenti se spacchi solo negli intenti"
            },
            {
              "start": 81050,
              "value": " E crei pochi smottamenti"
            },
            {
              "start": 82800,
              "value": " A capirci, sbattimenti per gli utenti"
            },
            {
              "start": 84840,
              "value": " Ma poi quando lo fanno se ne vanno via contenti (Ciao)"
            },
            {
              "start": 87520,
              "value": " D'altro canto, canto altro, altro smalto"
            },
            {
              "start": 89380,
              "value": " Altro stampo, altra stampa, ma non campo"
            },
            {
              "start": 91520,
              "value": " Dritto nell'impianto da schianto schiantati"
            },
            {
              "start": 93730,
              "value": " Puoi colpire al fianco, ma tanto non sfianchi qui"
            },
            {
              "start": 96430,
              "value": " Quindi anche se ci mancano i dindi"
            },
            {
              "start": 98440,
              "value": " No, a noi non ci restringi"
            },
            {
              "start": 100120,
              "value": " Se t'eri fatto il quadro ridipingici"
            },
            {
              "start": 102240,
              "value": " Oh, ripigliati, dai stringi le meningi"
            },
            {
              "start": 104980,
              "value": " Meddaman e Goediman, ricordi"
            },
            {
              "start": 106810,
              "value": " Abbiamo un quesito da porti"
            },
            {
              "start": 108580,
              "value": " Tu da quale parte ti senti, di chi ha i sentimenti"
            },
            {
              "start": 111290,
              "value": " O di chi li usa solo come strumenti"
            },
            {
              "start": 113360,
              "value": " Scusi, signorina, lei lo sa come funziona?"
            },
            {
              "start": 115550,
              "value": " Certo che lo so, c'è il dj che li suona"
            },
            {
              "start": 117100,
              "value": " Medda e Geodi Microspasmi in zona"
            },
            {
              "start": 119940,
              "value": " Portano il viaggio nel villaggio e l'aria buona"
            },
            {
              "start": 121810,
              "value": " E non li fermi, non li fermi, non li fermano più"
            },
            {
              "start": 123100,
              "value": " Non li fermi anche se chiami mille macchine blu"
            },
            {
              "start": 126160,
              "value": " Porta il vero Microspasmi questa resta la crew"
            },
            {
              "start": 128320,
              "value": " Con la forma più la firma quando arrivano giù"
            },
            {
              "start": 130520,
              "value": " E non si pensi che si è spenti"
            },
            {
              "start": 133240,
              "value": " Abbiamo la materia ed occhi grossi densi"
            },
            {
              "start": 135080,
              "value": " E tu, tu che cosa pensi?"
            },
            {
              "start": 137140,
              "value": " Che alla fine resteremo solamente in centoventi"
            },
            {
              "start": 139580,
              "value": " Tu da quale parte ti senti, di chi ha i sentimenti"
            },
            {
              "start": 142150,
              "value": " O di chi li usa come strumenti"
            },
            {
              "start": 143850,
              "value": " Magari dentro il club non ci pensi"
            },
            {
              "start": 145490,
              "value": " Ma quando torni a casa a fari spenti stringi i denti"
            },
            {
              "start": 147790,
              "value": " Qui conviene che non rallenti se no tamponaus"
            },
            {
              "start": 150460,
              "value": " Qui a fretta con le renne anche Santa Claus"
            },
            {
              "start": 152640,
              "value": " Butta via il tuo mouse, una sera vieni sotto"
            },
            {
              "start": 154810,
              "value": " Così senti quanto e forte da vicino il botto"
            },
            {
              "start": 157030,
              "value": " Otto, sette, giorni, sei dei nostri"
            },
            {
              "start": 159870,
              "value": " Oppure dopo cinque già ci molli"
            },
            {
              "start": 161360,
              "value": " Crolli come frollini molli"
            },
            {
              "start": 162960,
              "value": " Oppure nella jam tiri fuori prima giolle e jolli"
            },
            {
              "start": 165710,
              "value": " Scusi, signorina, lei lo sa come funziona?"
            },
            {
              "start": 167960,
              "value": " Certo che lo so, c'è il dj che li suona"
            },
            {
              "start": 170300,
              "value": " Medda e Geodi Microspasmi in zona"
            },
            {
              "start": 172290,
              "value": " Portano il viaggio nel villaggio e l'aria buona"
            },
            {
              "start": 174270,
              "value": " E non li fermi, non li fermi, non li fermano più"
            },
            {
              "start": 176480,
              "value": " Non li fermi anche se chiami mille macchine blu"
            },
            {
              "start": 178580,
              "value": " Porta il vero Microspasmi questa resta la crew"
            },
            {
              "start": 180730,
              "value": " Con la forma più la firma quando arrivano giù"
            },
            {
              "start": 182930,
              "value": " E non si pensi che si è spenti"
            },
            {
              "start": 185580,
              "value": " Abbiamo la materia ed occhi grossi densi"
            },
            {
              "start": 187570,
              "value": " E tu, tu che cosa pensi?"
            },
            {
              "start": 189550,
              "value": " Che alla fine ci saranno grossi mutamenti"
            },
            {
              "start": 192070,
              "value": " "
            }
          ],
          "displayArtist": "Microspasmi",
          "displayTitle": "Non Li Fermano Piu'"
        }
      ]
    }
  }
}
```

**Comment by BLeeEZ** [comment]
I will take a look at it

**Comment by BLeeEZ** [comment]
Should be fixed with 952abaae4d0a585e7145910b6800f40468e854f9.
Could you please test on your side too and verify?

**Comment by danielepintore** [comment]
Hi thanks for the rapid response 😃, tested on my side too, the problem is solved.
